### ISSUE TITLE
Update Products after first import

### ISSUE THREAD

**Issue by damiansoriano** [openlabs/magento_integration#69]
Hi there!

I am using the Magento Integration module and I found something that may be a lacking functionality:

1- Create a product in Magento with name "Product 01".
2- Import it in OpenERP with magento-integration. This creates a "Product 01" product in OpenERP
3- Go to Magento and change name from "Product 01" to "Product 02".
4- Import again products with magento-integration to OpenERP. In OpenERP the product is still called "Product 01" instead of "Product 02".

Bottom line, products don't get updated from Magento to OpenERP, the only flow of information is when the products are first imported and not later.

Are you planning to add this functionality for updating products information after the first Import?

Damián Soriano

**Comment by shalabhaggarwal** [comment]
Hi Damián,

You are right about this.

We are planning the best way to do this and this feature should be in soon.

We will track the progress of this development on this issue itself.

Cheers!!

**Comment by damiansoriano** [comment]
Grate, I will test it later!

Thanks!
